demo/admin/src/news/generated/NewsGrid.tsx

```diff
@@ -0,0 +1,237 @@
+// This file has been generated by comet admin-generator.
+// You may choose to use this file as scaffold by moving this file out of generated folder and removing this comment.
+import { useApolloClient, useQuery } from "@apollo/client";
+import {
+    CrudContextMenu,
+    GridFilterButton,
+    muiGridFilterToGql,
+    muiGridSortToGql,
+    StackLink,
+    Toolbar,
+    ToolbarAutomaticTitleItem,
+    ToolbarFillSpace,
+    ToolbarItem,
+    useBufferedRowCount,
+    useDataGridRemote,
+    usePersistentColumnState,
+} from "@comet/admin";
+import { Add as AddIcon, Edit } from "@comet/admin-icons";
+import { BlockPreviewContent } from "@comet/blocks-admin";
+import { DamImageBlock } from "@comet/cms-admin";
+import { Box, Button, IconButton } from "@mui/material";
+import { DataGridPro, GridColDef, GridToolbarQuickFilter } from "@mui/x-data-grid-pro";
+import { useContentScope } from "@src/common/ContentScopeProvider";
+import gql from "graphql-tag";
+import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { NewsContentBlock } from "../blocks/NewsContentBlock";
+import {
+    GQLCreateNewsMutation,
+    GQLCreateNewsMutationVariables,
+    GQLDeleteNewsMutation,
+    GQLDeleteNewsMutationVariables,
+    GQLNewsGridQuery,
+    GQLNewsGridQueryVariables,
+    GQLNewsListFragment,
+} from "./NewsGrid.generated";
+
+const newsFragment = gql`
+    fragment NewsList on News {
+        id
+        updatedAt
+        slug
+        title
+        date
+        category
+        visible
+        image
+        content
+        createdAt
+    }
+`;
+
+const newsQuery = gql`
+    query NewsGrid($offset: Int, $limit: Int, $sort: [NewsSort!], $search: String, $filter: NewsFilter, $scope: NewsContentScopeInput!) {
+        newsList(offset: $offset, limit: $limit, sort: $sort, search: $search, filter: $filter, scope: $scope) {
+            nodes {
+                ...NewsList
+            }
+            totalCount
+        }
+    }
+    ${newsFragment}
+`;
+
+const deleteNewsMutation = gql`
+    mutation DeleteNews($id: ID!) {
+        deleteNews(id: $id)
+    }
+`;
+
+const createNewsMutation = gql`
+    mutation CreateNews($scope: NewsContentScopeInput!, $input: NewsInput!) {
+        createNews(scope: $scope, input: $input) {
+            id
+        }
+    }
+`;
+
+function NewsGridToolbar() {
+    return (
+        <Toolbar>
+            <ToolbarAutomaticTitleItem />
+            <ToolbarItem>
+                <GridToolbarQuickFilter />
+            </ToolbarItem>
+            <ToolbarFillSpace />
+            <ToolbarItem>
+                <GridFilterButton />
+            </ToolbarItem>
+            <ToolbarItem>
+                <Button startIcon={<AddIcon />} component={StackLink} pageName="add" payload="add" variant="contained" color="primary">
+                    <FormattedMessage id="news.newNews" defaultMessage="New News" />
+                </Button>
+            </ToolbarItem>
```

Comment:
Buttons to the right of the Toolbar should be wrapped by a single `ToolbarActions` instead of individual `ToolbarItem` components. 

 ```suggestion
            <ToolbarActions>
                <GridFilterButton />
                <Button startIcon={<AddIcon />} component={StackLink} pageName="add" payload="add" variant="contained" color="primary">
                    <FormattedMessage id="news.newNews" defaultMessage="New News" />
                </Button>
            </ToolbarActions>
```